docs/source/index.rst

```diff
@@ -11,9 +11,11 @@ Autometa documentation
 Quickstart
 ==========
 
-Run with below code::
+Run with below code:
 
-    python autometa.py metagenome.config
+.. code-block:: bash
+    
+    ./nextflow run autometa.nf
```

Comment:
`./nextflow` assumes the user has downloaded a nextflow executable into that directory. 

If the user has installed nextflow via conda (should be able to with `conda install -n autometa --file=requirements.txt`) the command will be available from the conda env, _not_ as an executable in the user's directory

```suggestion
    nextflow run autometa.nf
```